tests/command/commands/user_test.py

```diff
@@ -1,66 +1,174 @@
 """Test user command parsing."""
 from command.commands.user import UserCommand
+from unittest import mock
+from model.user import User
+from db.facade import DBFacade
+from model.permissions import Permissions
+from bot.bot import Bot
 
 
 def test_get_command_name():
     """Test user command get_name method."""
-    testcommand = UserCommand()
```

Comment:
A lot of this setup is shared between unit tests - this can be cleaned up a bit by using test cases/test fixtures, which are provided by both [pytest](
http://doc.pytest.org/en/latest/fixture.htm) and via [`unittest`](
https://docs.python.org/3/library/unittest.htm), which can be run by Pytest. I believe the [`bot` tests](
https://github.com/ubclaunchpad/rocket2.0/blob/master/tests/bot/bot_test.py) are already in the `unittest` style.